cmd/openshift-install/agent.go

```diff
@@ -130,6 +130,7 @@ var (
 			&tls.KubeAPIServerLocalhostSignerCertKey{},
 			&tls.KubeAPIServerServiceNetworkSignerCertKey{},
 			&tls.AdminKubeConfigSignerCertKey{},
+			&image.AgentPassword{},
```

Comment:
Maybe the short description could be modified to include also the password?

Reply:
Done